tasties_app/tests/test_comment.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from tasties_app.models import Comment, Recipe
+from django.contrib.auth.models import User
+from django.utils import timezone
+from datetime import date
+
+
+@pytest.fixture
+def generate_comments():
+    """
+    in this generate function we create User, Category and Recipe
+    objects to fill Comment class and used to test
+     """
+    user = User.objects.create_user(username='test_user', password='password')
+    recipe = Recipe(title='Test Recipe1', author_id=user, description='Test Description1',
+                    directions='Test Directions1', publication_date=timezone.now(), minutes_to_make=1,
+                    recipe_picture='recipe.jpeg')
+    recipe.save()
+    the_date = date.today().strftime("%Y-%m-%d")
+    comment = Comment(author_id=user, recipe_id=recipe, publication_date=the_date, comment_text='test comment')
```

Comment:
Here as well, we need to call `comment.clean_fields()` in a `try-catch` block before saving it.

Reply:
why I should make that? i don't understand . so the fields it will be empty... and the test will not work.